core/trino-main/src/main/java/io/trino/operator/FlatHashStrategyCompiler.java

```diff
@@ -352,6 +357,85 @@ private static void generateHashBlock(ClassDefinition definition, List<KeyField>
         body.append(result.ret());
     }
 
+    private static void generateHashBlocksVectorized(ClassDefinition definition, List<KeyField> keyFields, CallSiteBinder callSiteBinder)
```

Comment:
I'd name this "batched" instead of "vectorized"

Reply:
Renamed